lib/documents.js

```diff
@@ -37,7 +37,11 @@ function Paragraph(children, properties) {
         styleId: properties.styleId || null,
         styleName: properties.styleName || null,
         numbering: properties.numbering || null,
-        alignment: properties.alignment || null
+        alignment: properties.alignment || null,
+        indentLeft: properties.indentLeft || null,
+        indentRight: properties.indentRight || null,
+        indentFirstLine: properties.indentFirstLine || null,
+        indentHanging: properties.indentHanging || null
```

Comment:
It ensures consistency and makes it clear what properties you can expect to find on a paragraph, which is simpler for consumers to read than having to understand the parser.